src/content/code/plugin-api/layers/override-layer.ts

```
@@ -0,0 +1,68 @@
+    label: true,
+    labelText: "sample",
+    labelPosition: "top",
+    labelTypography: {
+      italic: true,
+    },
+  },
+};
+const layerId = reearth.layers.add(sampleLayer);
+
+//1. Example to set the visible of a layer to hidden
+reearth.layers.override(layerId, { visible: false });
```

Biome

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)